lib/Connector.js

```diff
@@ -151,44 +148,33 @@ var Connector = {
 
 		var apiWhitelist = ["segment_list", "tracking_event_remove", "contact_list", "form_html", "tracking_site_status", "tracking_event_status", "tracking_whitelist", "tracking_log", "tracking_site_list", "tracking_event_list"];
 
-		var api_request = new RSVP.Promise(function(fulfill, reject) {
-			request(request_options, function(error, response, body) {
+		var api_request = axios(request_options)
+			.then((response) => {
 				try {
-					var result = JSON.parse(body);
+					JSON.parse(body);
 				} catch (e) {
-					apiWhitelist.forEach(function(value) {
+					apiWhitelist.forEach(function (value) {

```

Comment:
I thought we have removed that already. Is that still supposed to be there?

Reply:
Yeah, I believe this whitelist works in a different way than the other class-specific whitelists. I don't fully understand how it's used yet so I'm not confident in removing it. Maybe future exploration will reveal that it is no longer necessary.